exercise.wwwapi/Endpoints/BookEndpoints.cs

```diff
@@ -0,0 +1,69 @@
+﻿
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http;
+using System.Threading.Tasks;
+using exercise.wwwapi.Models;
+using exercise.wwwapi.Repository;
+
+namespace exercise.wwwapi.Endpoints
+{
+    public static class BookEndpoints
+    {
+        public static void ConfigureBookEndpoint(this WebApplication app)
+        {
+            var bookGroup = app.MapGroup("");
+
+            bookGroup.MapPost("/create a book", AddBook);
```

Comment:
when making this call it will look like this.

localhost:1234/book/create a book

which is not correct. no need to add phrase on the roots

POST
localhost:1234/book